torch/csrc/inductor/aoti_model_runner_cuda.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_model_runner.h>
+
+namespace torch {
+namespace inductor {
```

Comment:
Please, we live in C++17 world :)
```suggestion
namespace torch::inductor {
```